packages/moonstone/Heading/Heading.module.less

```diff
@@ -0,0 +1,64 @@
+// Heading.module.less
+//
+@import "../styles/variables.less";
+@import "../styles/mixins.less";
+@import "../styles/skin.less";
+
+.heading {
+	border-bottom: @moon-heading-border-width solid transparent;
+	margin: 0 @moon-spotlight-outset (1.5 * @moon-spotlight-outset) @moon-spotlight-outset;
+	min-height: (1.5 * @moon-spotlight-outset);
```

Comment:
Maybe create heading variables for these as well.

Reply:
These are sort of the "generic minimum passable" rules, fallback rules if you will. The various sizes do have min-height and margin-bottom, which apply in the spacing section.